qlib/workflow/online/utils.py

```diff
@@ -12,6 +12,7 @@
 
 from qlib.log import get_module_logger
 from qlib.utils import get_cls_kwargs
+from qlib.utils.exceptions import QlibException
```

Comment:
`QlibException` looks too general
You can create a new exception class based on `QlibException` which is more concrete.